src/util/urlUtils.js

```diff
@@ -16,27 +16,32 @@ export const globalBannerDenyList = [
 	'/process-instant-lending/*',
 ];
 
+function matchRoute(route, pattern) {
```

Comment:
It looks like this file already has a unit testing file, consider adding new unit tests for these two methods.